packages/design-system-docs/src/scripts/components/Header.jsx

```diff
@@ -1,6 +1,6 @@
 import GitHubLinks from './GitHubLinks';
 import React from 'react';
-import UsaBanner from './UsaBanner';
+import { UsaBanner } from '@cmsgov/design-system';
```

Comment:
We can remove the props being passed to the component
        <UsaBanner
          isBannerOpen={this.state.isBannerOpen}
          onToggleBanner={this.handleToggleBanner}
        />

Reply:
and remove the state and handToggleBanner from here.